packages/aws-cdk/lib/init-templates/app/fsharp/src/HelloCdk/HelloStack.fs

```diff
@@ -17,5 +17,5 @@ type MyStack(scope, name, props) as this =
     let user = User(this, "MyUser", UserProps())
 
     do
-        topic.SubscribeQueue(queue) |> ignore
+        topic.SubscribeQueue(queue, Nullable false) |> ignore
```

Comment:
:-( we still don't have support for optional arguments in .NET?

Reply:
Was about to open a PR for these exact same changes, beat me to it. 
BTW I was using `Nullable(type)` instead of `Nullable type`, not fussed with one over the other (barely use Nullable in F# anyway) but should try to stick to one.